docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,65 @@
+# Using devices with Clear Containers
```

Comment:
Capitalize "Devices"

Reply:
@klynnrif please note sentence case is preferred in all CC headings.

Reply:
@iphutch Thanks! Got it.